widgets/SalesforceMultiObjects-batchsource.json

```diff
@@ -74,26 +74,52 @@
       "properties": [
         {
           "widget-type": "textbox",
-          "label": "Datetime Filter",
-          "name": "datetimeFilter",
+          "label": "Last Modified After",
+          "name": "datetimeAfter",
           "widget-attributes" : {
-            "placeholder": "Datetime filter in SOQL Date or Date Literal format"
+            "placeholder": "Datetime filter in SOQL Date format"
```

Comment:
I think it's more useful if this shows the user what they can type in. Something like:

YYYY-MM-DDThh:mm:ssZ (ex: 2019-01-01T00:00:00Z)